src/Driver/Pdo/LogTypes.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Db\Driver\Pdo;
+
+class LogTypes
+{
+    public const KEY = 'log-type';
```

Comment:
`LogTypes` is bad place for this constant. `LogTypes` by name should contain only types. May be move `KEY` to new class `LogContext::TYPE` ?

It allow simplify constants:

- `LogTypes::TYPE_CONNECTION` → `LogTypes::CONNECTION`
- ...

Also suggest rename class to `LogType`. `LogType::Connection` is more pretty.

Reply:
About rename class and simplify constants - good idea. New class for one constatns - KEY, it's overhead. I'm simple rename const KEY to LOG_TYPE.